homeassistant/components/knx/schema.py

```diff
@@ -210,6 +214,9 @@ class ConnectionSchema:
     CONF_KNX_ROUTE_BACK = "route_back"
     CONF_KNX_STATE_UPDATER = "state_updater"
 
+    CONF_KNX_DEFAULT_STATE_UPDATER = True
+    CONF_KNX_DEFAULT_RATE_LIMIT = 20
```

Comment:
move to const.py too - at one point these are not needed here anymore when the deprecated schema is removed

Reply:
I'd remove all of this in ~3 months when we can deprecate it